components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dto/OAuthConsumerAppDTO.java

```diff
@@ -34,6 +34,7 @@
     private boolean pkceSupportPlain;
     private boolean pkceMandatory;
     private String state;
+    private long oauthExpiryTime;
```

Comment:
would be better to use a name like clientSecretExpireAt or something that indicates its original idea

Reply:
This attribute has been removed with the latest changes